test/jdk/java/awt/FullScreen/FullScreenInsets/FullScreenInsets.java

```diff
@@ -21,45 +21,53 @@
  * questions.
  */
 
+import javax.imageio.ImageIO;
 import java.awt.AWTException;
 import java.awt.Color;
 import java.awt.Dimension;
-import java.awt.DisplayMode;
 import java.awt.Frame;
 import java.awt.GraphicsDevice;
 import java.awt.GraphicsEnvironment;
 import java.awt.Insets;
+import java.awt.Rectangle;
 import java.awt.Robot;
 import java.awt.Window;
 import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
 
-/**
+/*
  * @test
  * @key headful
  * @bug 8003173 7019055
  * @summary Full-screen windows should have the proper insets.
- * @author Sergey Bylokhov
+ * @run main FullScreenInsets
  */
 public final class FullScreenInsets {
 
     private static boolean passed = true;
     private static Robot robot = null;
+    private static int deviceCount = 0;
+    private static final float TOLERANCE = 10;
```

Comment:
Are you sure that we need so big tolerance just to capture the green and red colors? We have CheckCommonColors which checks all our common colors via robot, and seems it works fine w/o tolerance.

Reply:
I see your point. I will recheck using `CheckCommonColors` test on all platforms and reduce the tolerance to a lower value so that it has more strict constraint on allowed colors.